src/irr/video/COpenGLFunctionTable.h

```diff
@@ -19,3116 +19,5702 @@
+				const void* data)
+			{
+				glBuffer.pglClearNamedBufferSubData(buffer, internalformat, offset, size, format, type, data);
+			}
+			inline void GlClearNamedBufferSubDataEXT(GLuint buffer,
+				GLenum internalformat,
+				GLintptr offset,
+				GLsizeiptr size,
+				GLenum format,
+				GLenum type,
+				const void* data)
+			{
+				glBuffer.pglClearNamedBufferSubDataEXT(buffer, internalformat, offset, size, format, type, data);
+			}
+
+			/// <summary>
+			/// copy all or part of the data store of a buffer object to the data store of another buffer object
+			/// </summary>
+			inline void GlCopyBufferSubData(GLenum readTarget,
+				GLenum writeTarget,
+				GLintptr readOffset,
+				GLintptr writeOffset,
+				GLsizeiptr size)
+			{
+				glBuffer.pglCopyBufferSubData(readTarget, writeTarget, readOffset, writeOffset, size);
+			}
+
+			/// <summary>
+			/// copy all or part of the data store of a buffer object to the data store of another buffer object
+			/// </summary>
+			inline void GlCopyNamedBufferSubData(GLuint readBuffer,
+				GLuint writeBuffer,
+				GLintptr readOffset,
+				GLintptr writeOffset,
+				GLsizeiptr size)
+			{
+				glBuffer.pglCopyNamedBufferSubData(readBuffer, writeBuffer, readOffset, writeOffset, size);
+			}
+			inline void Gl(GLuint readBuffer,
+				GLuint writeBuffer,
+				GLintptr readOffset,
+				GLintptr writeOffset,
+				GLsizeiptr size)
+			{
+				glBuffer.pglNamedCopyBufferSubDataEXT(readBuffer, writeBuffer, readOffset, writeOffset, size);
+			}
+			inline GLboolean GlIsBuffer(GLuint buffer)
+			{
+				return glBuffer.pglIsBuffer(buffer);
+			}
+			/// <summary>
+			/// return parameters of a buffer object
+			/// </summary>
+			inline void GlGetNamedBufferParameteri64v(GLuint buffer, GLenum pname, GLint64* value)
+			{
+				glBuffer.pglGetNamedBufferParameteri64v(buffer, pname, value);
+			}
+
+			/// <summary>
+			/// return parameters of a buffer object
+			/// </summary>
+			inline void GlGetBufferParameteri64v(GLenum target, GLenum pname, GLint64* value)
+			{
+				glBuffer.pglGetBufferParameteri64v(target, pname, value);
+			}
+
+			/// <summary>
+			/// return parameters of a buffer object
+			/// </summary>
+			inline void GlGetNamedBufferParameteriv(GLuint buffer,
+				GLenum pname,
+				GLint* params)
+			{
+				glBuffer.pglGetNamedBufferParameteriv(buffer, pname, params);
+			}
+			inline void GlGetNamedBufferParameterivEXT(GLuint buffer,
+				GLenum pname,
+				GLint* params)
+			{
+				glBuffer.pglGetNamedBufferParameterivEXT(buffer, pname, params);
+			}
+			inline void GlGetBufferParameteriv(GLenum target,
+				GLenum value,
+				GLint* data)
+			{
+				glBuffer.pglGetBufferParameteriv(target, value, data);
+			}
+
+
+			// GL Texture
+
+			/// <summary>
+			/// 
+			/// </summary>
+			inline void GlActiveTexture()
+			{
+				glTexture.pglActiveTexture();
+			}
+			/// <summary>
+			/// 
+			/// </summary>
+			inline void GlBindTextures()
+			{
+				glTexture.pglBindTextures();
+			}
```

Comment:
I don't think I want these public, nor their pointers

Reply:
Should i use the macro to create another class with pointers again but this time keep the class private?